src/ui/qgsprojectpropertiesbase.ui

```diff
@@ -1442,6 +1449,19 @@
              </layout>
             </widget>
            </item>
+           <item row="5" column="0">
+            <spacer name="verticalSpacer_2">
+             <property name="orientation">
+              <enum>Qt::Vertical</enum>
+             </property>
+             <property name="sizeHint" stdset="0">
+              <size>
+               <width>20</width>
+               <height>40</height>
```

Comment:
Wouldn't that vertical spacer take space for nothing? Or is the widget collapsible?
I mean, on small screens or while trying to take optimal sized screenshots, the spacer at the bottom sometimes oblige to enlarge the dialog if you want to show more items in the frame. My preference goes to  a height of 0.

Reply:
well, it seems equivalent to removing the spacer, which I've done.